### ISSUE TITLE
Suggestion: Reply threads

### ISSUE THREAD

**Issue by Mattis142** [mastodon/mastodon-android#264]
Would it be possible to implement reddit-style "reply threads" which make it more obvious who is replying to what, while also saving space below a toot

![Screenshot_20221028-174115](https://user-images.githubusercontent.com/40364718/198678077-edd35a7e-2f0c-4826-8bea-cdeb88a59680.png)
![Screenshot_20221028-174046](https://user-images.githubusercontent.com/40364718/198678097-c6759d31-db5e-49a2-ac3f-7ed81eacd5ce.png)

**Comment by nemobis** [comment]
Do you mean indentation?

**Comment by MKCOOL142** [comment]
> Do you mean indentation?

Yeah I guess, I just find that structure a lot easier to understand

**Comment by realpixelcode** [comment]
Honestly, that looks horrible. You can use Fedilab if you insist on thread indentation.

**Comment by juancnuno** [comment]
The Android app should at least be consistent with the Web client
![Screenshot from 2024-10-01 17-00-37](https://github.com/user-attachments/assets/276bcac7-2812-4be8-ba3a-83fc2d655b08)
![Screenshot_20241001-170142](https://github.com/user-attachments/assets/6eb32b48-c9a4-4ba6-bcac-78f8289ad70e)
